src/units/PhysicalConstants.hh

```diff
@@ -48,7 +49,25 @@ namespace rtt_units {
 class PhysicalConstants {
 public:
   // Constructors.
-  PhysicalConstants();
+  //----------------------------------------------------------------------------// 
```

Comment:
It seems there is a style problem here (apologies) - it may need to have clang-format run.

Reply:
fixed in a6db606